File: wear/src/main/java/hibernate/v2/testyourandroid/ui/activity/BaseActivity.java
package hibernate.v2.testyourandroid.ui.activity;

import android.annotation.SuppressLint;
import android.app.Activity;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

/**
 * Created by himphen on 25/5/16.
 */
@SuppressLint("Registered")
public class BaseActivity extends AppCompatActivity {

	protected Activity mContext;

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		mContext = this;
	}
}

File: wear/src/main/java/hibernate/v2/testyourandroid/ui/base/BaseActivity.kt
package hibernate.v2.testyourandroid.ui.base

import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import hibernate.v2.testyourandroid.R

/**
 * Created by himphen on 25/5/16.
 */
abstract class BaseActivity : AppCompatActivity() {

    fun initFragment(fragment: Fragment) {
        setContentView(R.layout.activity_container_no_drawer)
        supportFragmentManager.beginTransaction()
                .replace(R.id.container, fragment)
                .commit()
    }
}